plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -883,7 +883,20 @@ SecRule REQUEST_FILENAME "@endsWith /wp-admin/admin-ajax.php" \
 
 SecMarker "END-WORDPRESS-ADMIN"
 
-
+# Fix FP preventing access to plugins|plugin-install pages
+# Tested with the CRS rules from default branch with tag - v4.0.0-rc1
+SecRule REQUEST_FILENAME "@rx /wp-admin/(?:plugins|plugin-install).php" \
```

Comment:
Yeah sure, I also thought about that. But [CRS Docs](https://coreruleset.org/docs/rules/creating/#advanced-operator-usage) explicitly mentioned avoiding usage of `^` and `$`. Will do as you say. (:

Reply:
I think this does not apply for `REQUEST_FILENAME` as adding any symbols before/after will prevent the attack in 99,9% of situations (as it will change the file name i. e. the target of the attack). Adding `^` and/or `$` will singificantly reduce the CPU time needed to process the regex. Thank you.

Reply:
Hey @azurit , what do you think about this?

```
SecRule REQUEST_FILENAME "@rx /wp-admin/(?:plugins|plugin-install)\.php$" \
     "id:9507951,\
     phase:4,\
     pass,\
     t:none,\
     nolog,\
     ver:'wordpress-rule-exclusions-plugin/1.0.1'\
     chain"
     SecRule RESPONSE_BODY "@rx (?i)The function" \
          "ctl:ruleRemoveById=953100"
```